spring-integration-core/src/main/java/org/springframework/integration/endpoint/AbstractPollingEndpoint.java

```diff
@@ -111,6 +114,21 @@ public void setTransactionSynchronizationFactory(TransactionSynchronizationFacto
 		this.transactionSynchronizationFactory = transactionSynchronizationFactory;
 	}
 
+	/**
+	 * Return the default error channel if the error handler is explicitly provided and
+	 * it is a {@link MessagePublishingErrorHandler}.
+	 * @return the channel or null.
+	 * @since 4.3
+	 */
+	public MessageChannel getDefaultErrorChannel() {
+		if (!this.errorHandlerIsDefault && this.errorHandler instanceof MessagePublishingErrorHandler) {
+			return ((MessagePublishingErrorHandler) this.errorHandler).getDefaultErrorChannel();
+		}
+		else {
+			return null;
```

Comment:
1. The default MPEH lazily loads the default `errorChannel` in the `defaultErrorChannel` field so the graph could change at runtime.
2. Every poller routes to the default `errorChannel` flow by default; it would be difficult to render that in a diagram if more than a few pollers are present.
3. I don't think it's too hard for people to understand that every diagram will show a "detached" flow starting with `errorChannel` which applies to all pollers that don't have an explicit channel attached; it could be rendered something like this....

```
Endpoint with explicit error flow

     ______
    |      |
    |     o|----- ...
    |      |
    |__o___|
       |
       |_______ ...
        


Endpoint with default error flow

     ______
    |      |
    |     o|----- ...
    |      |
    |___o__|
        |
    (default)




    
                              _______
                             |logging|
     _________________       |       |
    (___errorChannel_()------|o      |
                             |_______|
```